ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -56,7 +62,7 @@ const FreqSummary = React.memo((props) => {
   const { field, fieldTitle, variant, urls, queryParams, acDisplay, titleContainer, precision = 2 } = props
   const { populations = {}, chrom } = variant
   const population = populations[field] || {}
-  if (population.af === null || population.af === undefined) {
+  if (population.af === null || population.af === undefined) { //When population is "gnomad_svs", how can I remove the frequency for null "gnomad_svs_AF"?
```

Comment:
When `gnomad_svs_AF` is null, it is parsed as `0` after search. Therefore, the `population.af` is `0`, neither `null` nor `undefined`. So, it always displays `gnomAD SVs 0.0` which doesn't look right. Should I remove the display when `gnomad_svs_AF` is null or doesn't exist?

Reply:
so it should only be 0 for indices that have that field populated for some variants, as based on the index mapping. So in an exome SV index or a SNV index, this should be left out entirely/ null. However, if there are any variants that have this populated (like in a genome SV index) than we do expect this to be 0! The fact that some variants in the index are annotated with gnomad SVs means we tried to annotatate everything in that index, and if we tried and there wasn't anything that means its not in gnomad, and has an AF of 0. In other words, what you are seeing is the expected behavior